lib/sparql.jison

```diff
@@ -676,7 +685,7 @@ SPACES_COMMENTS       (\s+|{COMMENT}\n\r?)+
 %%
 
 QueryOrUpdate
-    : Prologue ( Query /* [1] QueryUnit */ | Update? | Path ) EOF
+    : Prologue ( Query /* [1] QueryUnit */ | Update? | Path | PathsQuery) EOF
```

Comment:
@psvkaushik move this to be a peer to SELECT, CONSTRUCT, DESCRIBE, and ASK in `Qry`.